### ISSUE TITLE
Support csproj

### ISSUE THREAD

**Issue by phrohdoh** [JosephWoodward/VSCodeILViewer#9]
`project.json` is on its way out of the door while (the newly-improved) `csproj` is becoming the default.

**Comment by josephwoodward** [comment]
Yes, this is on my list of things to do over the next few days. Support for both project.json and .csproj should hopefully be in the next release.

**Comment by josephwoodward** [comment]
This is a real blocker right now, MSBuildWorkspaces does not support .NET Standard and there's no other approach, so I'm not sure how I'm going to tackle this. Could really do with some advice from someone with a far greater knowledge of Roslyn than me.

**Comment by phrohdoh** [comment]
Why does .NET Standard need to be supported by a tool that automates compilation of a project then runs ildasm?

**Comment by josephwoodward** [comment]
Unfortunately it's a little more complicated than what (not to mention ildasm is Windows only, where as this extension works on all platforms). What's actually happening is under the bonnet the extension is using Roslyn to compile just the file you're trying to extract the IL from.

In order to this it uses the workspace to extracting all that file's dependencies in the process.

I'm sure there's a way around it, I just need to investigate how.

**Comment by josephwoodward** [comment]
I've actually got this open as a [StackOverflow question](http://stackoverflow.com/questions/42395336/roslyn-workspace-for-net-cores-new-csproj-format) too.

**Comment by phrohdoh** [comment]
What I don't understand is why you need .NET Standard support to use the MSBuild Roslyn workspace. Do you have a gitter or IRC channel?

**Comment by josephwoodward** [comment]
Because the extension loads the project then includes all references (dlls, NuGet packages) so all types are available. I can do that manually but the workspace API makes it easy. It's possible that there's another way that I'm not aware of? Do you know something I don't?

**Comment by josephwoodward** [comment]
Here's a GitHub issue about the discussion of making the MSBuildWorkspace support .NET Standard:
https://github.com/dotnet/roslyn/issues/17439
TLDR; it's a little while away. In the mean time the extension doesn't work with .csproj :(

**Comment by cchexcode** [comment]
As the other Issues are closed now or their state is undeterminable due to them being open since 2017, could you Kindly provide any information when or if the .csproj support will be continued?
That would be awesome since I'd like to use your extension for working on my bachelor thesis.
